client/src/modules/home/home.js

```diff
@@ -9,19 +10,31 @@ import './home.css';
 
 class HomeController {
   constructor(PlanService) {
+
     this.name = 'Your Plans';
     this.PlanService = PlanService;
+
     this.plans = [
```

Comment:
not an issue for now, but I typically see constant testing data declared outside the component and simply referenced in the constructor, give you a slightly cleaner / closer to the future look for inside the controller itself